langs/fr/common.json

```diff
@@ -20,43 +20,43 @@
-  "Animation List": "Liste des animations",
+  "Animation List": "Liste des séries et films d'animation",
   "Studios": "Studios",
-  "More animation releases": "Plus d'animations",
-  "Get Lucky": "Soyez chanceux",
+  "More animation releases": "Plus de séries et films d'animation",
+  "Get Lucky": "Je suis chanceux",
   "Random Pull": "Aléatoire",
-  "Soundtrack Count": "Nombres de oundtrack totales",
-  "Highlight Soundtrack": "Soundtrack sélectionnée",
-  "Game Count": "Nombre de soundrack de jeux vidéos",
-  "Animation Count": "Nombre de soundrack d'animations",
-  "Game Soundtracks": "Bande Originale de jeux vidéos",
-  "Animation Soundtracks": "Bande Originale d'animations",
-  "Email_Sent": "An email has been sent to the address linked to the account. Check your spam folder",
-  "Username or email": "Username or email",
-  "Recover password": "Recover password",
-  "Invalid_Login": "Invalid username/email or password",
-  "Username": "Username",
-  "Password": "Password",
-  "Failed_Recover": "Failed to recover password",
-  "Animations": "",
-  "Favorite_Added": "",
-  "Favorite_Removed": "",
-  "Favorite_Add": "",
-  "Favorite_Remove": "",
-  "Favorite_Login": "",
-  "Release Date": "",
-  "Artists": "",
-  "Classification": "",
-  "Game Soundtrack": "",
-  "Animation Soundtrack": "",
-  "Published by": "",
-  "Favorite Score": "",
-  "Check album at": "",
-  "Buy_Original": "",
-  "Download": "",
-  "Related Soundtracks": "",
-  "Become_Donator": "",
-  "Direct": "",
-  "Tracklist": "",
-  "Disc": ""
+  "Soundtrack Count": "Nombre total de pistes",
+  "Highlight Soundtrack": "Bande originale sélectionnée",
+  "Game Count": "Nombre de bandes originales de jeux vidéos",
+  "Animation Count": "Nombre de bandes originales de séries et films d'animation",
+  "Game Soundtracks": "Bandes originales de jeux vidéos",
+  "Animation Soundtracks": "Bandes originales de séries et films d'animation",
+  "Email_Sent": "Un email a été envoyé à l'adresse mail lié à ce compte. Vérifier vos spams.",
+  "Username or email": "Nom d'utilisateur ou adresse mail",
+  "Recover password": "Mot de passe oublié",
+  "Invalid_Login": "Nom d'utilisateur/adresse mail ou mot de passe invalide",
+  "Username": "Nom d'utilisateur",
+  "Password": "Mot de passe",
+  "Failed_Recover": "Echec de la procédure de récupération du mot de passe.",
+  "Animations": "Séries et films d'animation",
+  "Favorite_Added": "Ajouté à vos favoris.",
+  "Favorite_Removed": "Supprimé de vos favoris",
+  "Favorite_Add": "Ajouter aux favoris",
+  "Favorite_Remove": "Supprimer des favoris",
+  "Favorite_Login": "Connectez vous pour l'ajouter à vos favoris",
+  "Release Date": "Date de sortie",
+  "Artists": "Artistes",
+  "Classification": "Classification",
+  "Game Soundtrack": "Bande originale de jeux-vidéos",
+  "Animation Soundtrack": "Bande originale de séries et films d'animation",
+  "Published by": "Publié par",
+  "Favorite Score": "Bande originale préférée",
```

Comment:
Not the preferred Soundtrack

Reply:
I didn't find where the "Favorite Score" is used. I translated according the other translations. If you can give me extra information, I will improve the translation.

Reply:
I think it's the number of time that a soundtrack is added to the favorites (in spanish it's translated "Popularidad", so I think it can be "Popularité" or "Nombre d'ajout aux favoris")

Reply:
Portuguese translations are more literal ("Pontuação Favorita"). Without context, it's not possible to give the correct translation.

Reply:
yeah, I'll ask